microsoft-edge/progressive-web-apps-chromium/how-to/index.md

```diff
@@ -418,7 +418,7 @@ To learn more about installing PWAs, see [Use Progressive Web Apps in Microsoft
 <!-- ====================================================================== -->
 ## Next steps
 
-The simple temperature converter PWA you built so far only scratches the surface of what PWAs can do. The previous steps are important prerequisites for any PWA, but there are important best practices that will make your PWA feel like a real app when installed.
+The temperature converter PWA you built so far only scratches the surface of what PWAs can do. The previous steps are important prerequisites for any PWA, but there are important best practices that will make your PWA feel like a real app when installed.
```

Comment:
debate: "scratches the surface" is a colloquialism, vs. corp style guide.  is there a nice direct wording?

Reply:
found it: 
https://learn.microsoft.com/en-us/style-guide/global-communications/writing-tips#tips-for-all-global-content - "Avoid idioms, colloquial expressions, and culture-specific references. They can be confusing for non-native English speakers and hard to localize. Consider the worldwide implications of what you write. Customers in other locales may not know much about the history and culture of your country."

Reply:
```
The temperature converter PWA that you built so far is only a 
small sample of what PWAs can do.
```

Reply:
As a non native English speaker, I can only agree with this. Let's not use "scratch the surface". Your proposal sounds good.